web-client/src/applicationContext.ts

```diff
@@ -624,6 +624,22 @@ const applicationContext = {
   convertBlobToUInt8Array: async blob => {
     return new Uint8Array(await new Response(blob).arrayBuffer());
   },
+  createCsvString: (
+    data: any[],
+    config: { displayLabel: string; key: string }[],
+  ) => {
+    const headers = config.map(c => `"${c.displayLabel}"`).join();
```

Comment:
It may be worth escaping certain characters that could break the CSV format like quotation marks inside the functions passed to the map method calls here.

Reply:
Good Point.

Right now we have 3 csv libraries and none of them can be ran in jest with the js-dom environment which is frustrating, so we decided to just write our own. Looking back I don't think that is a good idea either. I think we can convert to one of our other patterns (which is not testing it) but using a standard library to achieve this.

Ultimately I would like to see us converge on a single csv library that works on frontend/backend and can be run in the context of a test to reduce this nonsense.